### PR TITLE
Git 32 repurpose

### PR DESCRIPTION
**Pull request by eacharles**
Moved README.md aside to docs/BootCamp.md and replaced it with docs/GettingStarted.md.  

The basically changes to the doc from being focus on the BootCamp to being a more general resource for incoming people.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,104 @@
...
+to connect terminal and graphics to remote (i.e., SLAC / Stanford)
+computers.   It is also very likely that you will need to be able to
+run Jupyter analysis notebooks both on your computer, and by
+connecting your computer to a notebook server running at SLAC.
+
+[MacOS](docs/macos_setup.md)
+
+[Windows](docs/windows_setup.md)
+
+As usual, if you are using a linux computer you are on you own.
```
Comment (sibirrer) [inline README.md]: I wloud write it a bit more optimistic :), maybe state that you (the reader) can add resources if (s)he encounters problems.

File: docs/rubin_camera.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 To analyze test data from the LSST camera you will need a set of
 software tools known as the LSST Data Management (DM) stack, as well
-as a few other tools.  These tools are largely set up for you in the 
+as a few other tools.  These tools are largely set up for you already are easy to activate.
```
Comment (sibirrer) [inline docs/rubin_camera.md]: wording

**Review by sibirrer — COMMENTED** [review]
Thanks for setting this up. I only have very minor comments

File: docs/rubin_camera.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 To analyze test data from the LSST camera you will need a set of
 software tools known as the LSST Data Management (DM) stack, as well
-as a few other tools.  These tools are largely set up for you in the 
+as a few other tools.  These tools are largely set up for you already and easy to activate.
```
Comment (abmantz) [inline docs/rubin_camera.md]:
```suggestion
as a few other tools.  These tools are largely set up for you already and are easy to activate.
```

**Review by abmantz — APPROVED** [review]

**Pull request merged**